Source/Wikitivity.PreInstallInstanceSetting/PreInstallEventHandler.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Net;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using kCura.EventHandler;
+using kCura.EventHandler.CustomAttributes;
+using kCura.Relativity.Client;
+using Relativity.API;
+using Relativity.Services.Interfaces.InstanceSetting;
+using Relativity.Services.Interfaces.InstanceSetting.Model;
+using Relativity.Services.Objects;
+
+namespace Wikitivity.PreInstallInstanceSetting
+{
+	[RunOnce(true)]
+	[RunTarget(kCura.EventHandler.Helper.RunTargets.Workspace)]

```

Comment:
This creates an instance setting which is at ADMIN.  this should target Instance not workspace.  This will currently run on every workspace, which is wasting resources, when it only needs to run once and at admin mode.